server/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/ark-network/ark/server
 go 1.23.1
 
 replace github.com/ark-network/ark/common => ../common
+replace github.com/ark-network/ark/pkg/client-sdk => ../pkg/client-sdk
```

Comment:
these two statements shoudl not be needed actually! https://go.dev/blog/get-familiar-with-workspaces

Reply:
yeah, @louisinger drop it and run `go work sync` from the root folder of the project instead.